packages/web/app/src/lib/kit/helpers.ts

```diff
@@ -8,3 +8,20 @@ export const tryOr = <$PrimaryResult, $FallbackResult>(
     return fallback();
   }
 };
+
+export const oneOf = <type extends readonly unknown[]>(
```

Comment:
nit: Please add a description block to make the behavior more obvious when we use this function.
When seeing it used, I wasnt 100% sure it threw, or what type of error it would throw.